tests/.coveragerc

```diff
@@ -1,16 +1,43 @@
 [run]
 branch = True
 concurrency = multiprocessing,thread
-data_file = ${RUNTESTS_DIR-.}/.coverages/.coverage
+data_file = ${RUNTESTS_DIR-.}/.coverage
```

Comment:
We need to be careful if this would break the existing coverage CI job (https://djangoci.com/job/django-coverage/)
The easiest way to be sure we haven't broken this would to not really have changes to this config file

Reply:
reverted the whole thing back as it wasn’t worth the risk.